### PR TITLE
Fix employee table in all pages

### PR DESCRIPTION
**Pull request by marthaklambert**

### CODE REVIEW

File: client/src/pages/CreateSurvey/RecipientsList/index.jsx [hunk 1]
```diff
@@ -37,19 +50,44 @@ const RecipientsList = () => {
...
-            <EmployeesTable />
-            <Button
-              variant='contained'
-              color='secondary'
-              onClick={handleSaveRecipientsClick}
+        <Paper
+          style={{
+            width: '90vw',
+            height: '78vh',
+            // overflowX: 'auto',
```
Comment (Joaoviana) [inline client/src/pages/CreateSurvey/RecipientsList/index.jsx]: remove comment

File: client/src/pages/Dashboard/components/AllSurveysTable.jsx [hunk 2]
```diff
@@ -53,7 +55,10 @@ const AllSurveysTable = ({ surveys, history }) => {
               <TableRow
                 key={title}
                 className={styles.row}
-                onClick={() => history.push(`admin/surveys/${_id}`)}
+                onClick={() => {
+                  dispatch({ type: 'RESET_EMPLOYEE_DATA' });
```
Comment (Joaoviana) [inline client/src/pages/Dashboard/components/AllSurveysTable.jsx]: nice

File: server/dummyData/dummySurveys.js [hunk 3]
```diff
@@ -249,38 +249,10 @@ module.exports = [
     dateClosed: 1576851113,
     anonymous: true,
     recipients: [
```
Comment (Joaoviana) [inline server/dummyData/dummySurveys.js]: why was this needed?

Comment (marthaklambert) [reply server/dummyData/dummySurveys.js]: We only had one survey that didn't invite everyone which was why I didn't see the bug until today, and the title of the survey was "people team engagement survey" so seemed like it would be more sensible to just invite the people team.

Comment (Joaoviana) [reply server/dummyData/dummySurveys.js]: thats great, thank you!
